Code/Framework/AzCore/AzCore/Settings/SettingsRegistryConsoleUtils.cpp

```diff
@@ -35,7 +35,7 @@ namespace AZ::SettingsRegistryConsoleUtils
             const auto setOutput = AZ::SettingsRegistryInterface::FixedValueString::format(
                 R"(Successfully set value at path "%s" into the global settings registry)" "\n",
                 combinedKeyValueCommand.c_str());
-            AZ::Debug::Trace::Output("SettingsRegistry", setOutput.c_str());
+            AZ::Debug::Trace::Instance().Output("SettingsRegistry", setOutput.c_str());
```

Comment:
Honestly this logic is just to output to the Console Window in the Editor or GameLauncher or stdout in command line applications.
Having to go through the Trace system is unfortunate.